src/radix_connect/well_known_client/models/dapp_definitions.rs

```diff
@@ -0,0 +1,55 @@
+use crate::prelude::*;
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct DappDefinitions {
+    #[serde(rename = "dApps")]
+    pub dapp_definitions: Vec<DappDefinition>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub callback_path: Option<String>,
```

Comment:
Should be `RCMCallbackPath` type? (as in  `Option<RCMCallbackPath>`)?

Reply:
ok, how I make serde be able to serialize/deserialize RCMCallbackPath. Do I need additional setup?

Reply:
Is this question still relevant? 

`RCMCallbackPath` has the same shape as `BIP39Passphrase`. Just add `Serialize, Deserialize` in the derive... and use `#[serde(transparent)]`. look at `BIP39Passphrase`.